impl/client/src/main/javascript/web/util/xss.js

```diff
@@ -142,8 +142,27 @@ define("common-ui/util/xss", ["common-ui/dompurify"], function(DOMPurify) {
      */
     setHtmlUnsafe: function(elem, unsafeHtml) {
       elem.innerHTML = unsafeHtml;
-    }
-  });
+    },
+
+    /**
+     * Set's an anchor HTML element to a given sanitized Url.
+     *
+     * @param url
+     */
+    sanitizeUrl: function(url) {
+      try {
+        const parsedUrl = new URL(url);
+        const allowedProtocols = ['http:', 'https:'];
+        if (!allowedProtocols.includes(parsedUrl.protocol)) {
+          return 'about:blank';
+        }
+        return parsedUrl.href;
+      } catch (e) {
+        return 'about:blank';
```

Comment:
Ideally, return the default `about:blank` a single time, at the end of the method if reaching there.